src/appleseed.python/bindblendermeshconverter.cpp

```diff
@@ -64,22 +66,51 @@ namespace
         char flag, transp;
         short mode, tile, unwrap;
     };
+
+    // Blender 2.8 structures.
+    struct MLoop
+    {
+        // Vertex index.
+        unsigned int v;
+        unsigned int e;
+    };
+
+    struct MLoopTri
+    {
+        // Vertex id's.
+        unsigned int tri[3];
+        unsigned int poly;
+    };
+
+    struct MLoopUV
+    {
+        float uv[2];
+        int flag;
+    };
+
+    struct MPoly
+    {
+        int loopstart;
+        int totloop;
+        short mat_nr;
+        char flag, pad;
+    };
 }
 
 
 //
-// The following function takes a series of pointers to Blender mesh data
-// and modifies the appleseed MeshObject entity.
+//  The following function takes a series of pointers to Blender mesh data
```

Comment:
😂

What I meant is that indenting the lines after `Arguments:` make sense, it's a list after all.

```
//
// Comment here.
//
// Arguments:
//
//   first arg, indented
//   second arg, indented
//
```

Sorry if I wasn't clear. It's a detail anyway.